spec/setup_acceptance_node.pp

```diff
@@ -7,11 +7,19 @@
 class { 'candlepin::repo':
   version => 'nightly',
 }
-
 group { 'foreman':
   ensure => present,
 }
 
+# Necessary for PostgreSQL EVR extension
```

Comment:
I ran into the same thing with an experiment. It probably isn't worth to pull in `pulpcore` just for `pulpcore::repo`, but it would have the benefit of not having to maintain it in multiple places.

Reply:
Yea, thats why I went this route rather than put another dependency on, and figure we will just solve this via dropping the RPM one day.

Reply:
Agreed, we should work to that goal. Another solution may be to move it to foreman-packaging. AFAIK Pulp doesn't actually use it anyway.